src/domains/pay-wallet-common/00_alerts.tf

```diff
@@ -0,0 +1,222 @@
+data "azurerm_api_management" "apim" {
+  name                = "${local.product}-apim"
+  resource_group_name = "${local.product}-api-rg"
+}
+
+resource "azurerm_resource_group" "rg_payment_wallet_alerts" {
+  count    = var.env_short == "p" ? 1 : 0
+  name     = "${local.project}-alerts-rg"
+  location = var.location
+  tags     = var.tags
+}
+
+data "azurerm_key_vault_secret" "monitor_payment_wallet_opsgenie_webhook_key" {
+  count        = var.env_short == "p" ? 1 : 0
+  name         = "payment-wallet-opsgenie-webhook-token"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_monitor_action_group" "payment_wallet_opsgenie" {
+  count               = var.env_short == "p" ? 1 : 0
+  name                = "PayWalletOpsgenie"
+  resource_group_name = azurerm_resource_group.rg_payment_wallet_alerts[0].name
+  short_name          = "PayWalletOps"
+
+  webhook_receiver {
+    name                    = "PayWalletOpsgenieWebhook"
+    service_uri             = "https://api.opsgenie.com/v1/json/azure?apiKey=${data.azurerm_key_vault_secret.monitor_payment_wallet_opsgenie_webhook_key[0].value}"
+    use_common_alert_schema = true
+  }
+
+  tags = var.tags
+}
+
+#Availability API payment wallet for IO app V1
+resource "azurerm_monitor_scheduled_query_rules_alert" "payment_wallet_for_io_availability_v1" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "payment-wallet-for-io-availability-alert-v1"
+  resource_group_name = azurerm_resource_group.rg_payment_wallet_alerts[0].name
+  location            = var.location
+
+  action {
+    #action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, azurerm_monitor_action_group.payment_wallet_opsgenie[0].id]
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
+    email_subject          = "[payment-wallet-for-IO V1] Availability Alert"
+    custom_webhook_payload = "{}"
+  }
+  data_source_id = data.azurerm_api_management.apim.id
+  description    = "Payment Wallet for IO - Availability less than 99% in the last 30 minutes"
+  enabled        = true
+  query = (<<-QUERY
+AzureDiagnostics
+| where url_s startswith 'https://api.platform.pagopa.it/io-payment-wallet/v1'
+| summarize
+    Total=count(),
+    Success=countif(responseCode_d < 500)
```

Comment:
TODO: add response time in availability success count query
TODO: add alert for ecommerce-for-app-io api's in eCommerce domain